examples/forest-brawl/scripts/brawler-controller.gd

```diff
@@ -47,10 +47,6 @@ func _ready():
 	
 	_snap_to_spawn()
 
-	# TODO: What if the RollbackSynchronizer had a flag for this?
-	# Wait a frame so Input has time to get its authority set
```

Comment:
And that's what I did :P

Reply:
Turns out this is not needed at all, we can just defer the `process_settings()` call in RollbackSynchronizer - running it in the next idle time, usually at the end of the current `_process()`.